entrypoint.sh

```diff
@@ -48,4 +48,31 @@ if [[ 1 -gt "$(cat $GITHUB_ENV | grep -c AWS_REGION)" ]]; then
     else
         echo "AWS_REGION=$AWS_REGION" >> "$GITHUB_ENV"
     fi
-fi
\ No newline at end of file
+fi
+
+DEFAULT_SONAR_PROJECT_KEY=$(echo "$GITHUB_REPOSITORY" | tr / _ )
+echo "DEFAULT_SONAR_PROJECT_KEY=$DEFAULT_SONAR_PROJECT_KEY"
+
+# Setting the project key and project name for sonar property values coming from dotnet app
+SONAR_PROJECT_KEY_INPUT=$SONARCLOUD_PROJECT_KEY
```

Comment:
Form where would these keys come SONARCLOUD_PROJECT_KEY, SONARCLOUD_PROJECT_NAME

Reply:
These are set in the workflows on dotnet projects. 

**ie https://github.com/variant-inc/myvariant-eventhandler/blob/b688afbec32d7ea1d6715e78ed684d83d166b7e2/.github/workflows/branch-sonar.yaml#L11**

Reply:
So it defaults to that but get overwritten if sonarproject property files doesn't exist.. which it shouldn't for dotnet

Reply:
Can we document this better @sdoguparti. Everything looks good otherwise.

Reply:
https://github.com/variant-inc/actions-dotnet. Should create a ticket to update actions-dotnet docs https://github.com/variant-inc/actions-dotnet and in confluence with sonarcloud  setup